lib/attrsets.nix

```diff
@@ -1887,6 +1924,35 @@ rec {
   */
   getDev = getOutput "dev";
 
+  /**
+    Get a package's `include` output.
+    If the output does not exist, fallback to `.dev`, then to `.out`, and then to the default.
+
+    # Inputs
+
+    `pkg`
+
+    : The package whose `include` output will be retrieved.
+
+    # Type
+
+    ```
+    getInclude :: Derivation -> String
+    ```
+
+    # Examples
+    :::{.example}
+    ## `lib.attrsets.getInclude` usage example
+
+    ```nix
+    getInclude pkgs.openssl
+    => "/nix/store/00000000000000000000000000000000-openssl-1.0.1r-dev"
+    ```
+
+    :::
+  */
+  getInclude = getOutput' [ "include" "dev" ];
```

Comment:
The "include" name is motivated by the name of the respective `outputX` bash variable: https://github.com/NixOS/nixpkgs/blob/e77f5a04778586b5b0a9ff4fc8507de0ca07cc10/pkgs/build-support/setup-hooks/multiple-outputs.sh#L52

The bash variable defaults to `dev` and I'm not changing that because it'd trigger a mass rebuild and then maybe still break some.